packages/common/src/api/reducer.ts

```diff
@@ -1,11 +1,13 @@
 import { combineReducers } from 'redux'
 
+import collectionApi from './collection'
 import relatedArtistsApi from './relatedArtists'
 import trackApi from './track'
 import userApi from './user'
 
 export default combineReducers({
   relatedArtistsApi,
   trackApi,
+  collectionApi,
```

Comment:
alpha sort

Reply:
i let the linter do its thing. maybe this rule should be added?

Reply:
I think this one it will keep whatever order you use, but either way is fine. I think it may affect the order in the redux debugger?